addons/grenades/functions/fnc_flashbangExplosionEH.sqf

```diff
@@ -46,7 +46,7 @@ if (hasInterface) then {
 private _affected = (ASLtoAGL _grenadePosASL) nearEntities ["CAManBase", 20];
 _affected = _affected - [ACE_player];
 {
-    if (local _x && {alive _x}) then {
+    if (local _x && {_x call EFUNC(common,isAwake)} && {!(_x call EFUNC(common,isPlayer))}) then {
```

Comment:
`_affected = _affected - [ACE_player];` and `local _x`
should already guarantee they are AI I think, have you seen this effect players?
either way, I don't see any harm in adding the check

Reply:
No, I haven't. It was an incomplete thought process that lead me to conclude this, but yea, the existing code is enough.